pj/src/views/contact/Step1.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+<div class="box is-primary is-outlined">
+  <div class="notification">
+      <p id="step">STEP 1</p>
+      <h5 class="title is-5 has-text-centered">お客様の情報を入力してください</h5>
+  </div>
+  <div class="container is-size-5">
+    <h5 class="subtitle is-5 is-primary has-text-info">- 性別 -</h5>
+    <input name='status' type='radio' id='male' value="男性" v-model="customer.gender">
+    <label for="male">男性</label>
+    <input name='status' type='radio' id='female' value="女性" v-model="customer.gender">
+    <label for="female">女性</label>
+    
+  <br><br>
+
+    <h5 class="subtitle is-5 has-text-info">- 生年月日 -</h5>
+    <select id="year" v-model="customer.birthyear">
+      <option v-for="y in years" :key="y" :value="y.year">{{ y.text }}</option>
+    </select>
+    <label for="year">年</label>
+    <select id="month" v-model="customer.birthmonth">
+      <option v-for="n in 12" :key="n">{{ n }}</option>
+    </select>
+    <label for="month">月</label>
+    <select id="day" v-model="customer.birthday">
+      <option v-for="n in days" :key="n">{{ n }}</option>
+    </select>
+    <label for="day">日</label>
+
+  </div>
+</div>
+
+<div class="container" align="center">
+  <button class="button is-primary is-size-5" @click="forward">次へ進む ></button>
+</div>
+
+</template>
+
+<script>
+export default {
+  name: 'Step1',
+  data () {
+    return {
+      customer: 
+        {gender: '', birthyear: '1992', birthmonth: '1', birthday: '1'}
```

Comment:
`{}`の両内側にはスペースを空けましょう。また、下記のように改行することが多いかと思います。（コード規約はプロジェクトによりますが。。）
```suggestion
        { gender: '', 
          birthyear: '1992', 
          birthmonth: '1', 
          birthday: '1' }
```

Reply:
Vue2/3のエラーについて、おそらく原因わかりました。Vue2の場合は、<template>タグ内に、Single Root Element縛りがあることが原因でのエラーでした。Vue3→2移行が実例としてほとんどないのであれば、今回の課題上は割愛できたら幸いです。

Reply:
> Vue3→2移行が実例としてほとんどないのであれば、今回の課題上は割愛できたら幸いです。

はい、OKです！